bats/docs.bats

```diff
@@ -618,6 +618,15 @@ SQL
     run dolt schema show dolt_docs
     [ "$status" -eq 1 ]
     [[ "$output" =~ "'dolt_docs' is not a valid table name" ]] || false
+    run dolt schema show
+    [ "$status" -eq 1 ]
+    [[ "$output" =~ "usage: dolt schema show" ]] || false
```

Comment:
Don't think this regex is necessary. When there are dolt_docs and no other table dolt schema show should error.

Reply:
No need to test the help message.

Reply:
thanks, removed the message assertion